src/plugins/dev_tools/public/dev_tool.ts

```diff
@@ -66,7 +67,7 @@ export class DevToolApp {
   constructor(
     id: string,
     title: string,
-    mount: App['mount'],
+    mount: AppMount,
```

Comment:
I had to make this change in order for TS to pass. It kept thinking I was trying to use the deprecated interface.